### PR TITLE
//media/renderers visibility for //media/starboard

### PR DESCRIPTION
**Pull request by hlwarriner**
Use of VideoOverlayFactory from //media/renderers was added to //media/starboard in https://github.com/youtube/cobalt/pull/4287.

b/377295011

### CODE REVIEW

**Comment by hlwarriner** [comment]
It looks like we should add a reviewer from `//media/OWNERS` - please let me know if anyone has a suggestion.

**Review by kaidokert — APPROVED** [review]
Looks harmless from here.

File: media/renderers/BUILD.gn [hunk 1]
```diff
@@ -2,11 +2,18 @@
...
+if (is_cobalt) {
+  import("//starboard/build/buildflags.gni")
+}
+
 source_set("renderers") {
   # Do not expand the visibility here without double-checking with OWNERS, this
   # is a roll-up target which is part of the //media component. Most other DEPs
   # should be using //media and not directly DEP this roll-up target.
   visibility = [ "//media" ]
+  if (is_cobalt && use_starboard_media) {
```
Comment (kaidokert) [inline media/renderers/BUILD.gn:14]: I think `use_starboard_media` may be redundant here, because `is_cobalt` always implies it, no ?

Comment (andrewsavage1) [reply media/renderers/BUILD.gn:14]: Right now `use_starboard_media` can be set independently of `is_cobalt`, if we want to get rid of it entirely let's have a separate conversation if we want to get rid of it

**Review by andrewsavage1 — APPROVED** [review]

**Comment by hlwarriner** [comment]
> It looks like we should add a reviewer from `//media/OWNERS` - please let me know if anyone has a suggestion.

@kaidokert @andrewsavage1, any thoughts on this?

**Comment by andrewsavage1** [comment]
> It looks like we should add a reviewer from `//media/OWNERS` - please let me know if anyone has a suggestion.

You mean from the chromium org? Why? I think these changes are good as they are

**Comment by hlwarriner** [comment]
> > It looks like we should add a reviewer from `//media/OWNERS` - please let me know if anyone has a suggestion.
> 
> @kaidokert @andrewsavage1, any thoughts on this?

Discussed offline.

**Comment by hlwarriner** [comment]
> > It looks like we should add a reviewer from `//media/OWNERS` - please let me know if anyone has a suggestion.
> 
> You mean from the chromium org? Why? I think these changes are good as they are

Resolved in other comment thread.

**Pull request merged**